packages/dear-imgui/ts/src/lib/components/TradingGuiDemo/TradingGuiDemo.tsx

```
@@ -400,76 +400,76 @@ export const TradingGuiDemo = () => {
                     )}
-                </ReactImgui.Node> */}
-            </ReactImgui.Node>
-            <ReactImgui.Node style={styleSheet.sidebarNode}></ReactImgui.Node>
-
-            <ReactImgui.Node style={styleSheet.debugButton}>
-                <ReactImgui.Button label={faIconMap.bug} onClick={debugModeBtnClicked} />
-            </ReactImgui.Node>
-        </ReactImgui.Node>
+                </XFrames.Node> */}
+            </XFrames.Node>
+            <XFrames.Node style={styleSheet.sidebarNode}></XFrames.Node>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)